tests/io/test.base.js

```diff
@@ -8,11 +10,11 @@ describe('io.IOBase()', function() {
 
   it('should init class props as expected', () => {
     var io = new IOBase('foo/bar');
-    assert.equal(io.path, 'foo/bar');
-    assert.equal(io.entries.length, 0);
-    assert.equal(Object.keys(io.files).length, 0);
-    assert.equal(typeof io.files, 'object');
-    assert.equal(io.maxSizeBytes, 104857600);
+    expect(io.path).toEqual('foo/bar');
+    expect(io.entries.length).toEqual(0);
+    expect(Object.keys(io.files).length).toEqual(0);
```

Comment:
There's an open thread here: https://github.com/mozilla/addons-frontend/pull/2540/files/9ecf8b9664f359e27c0269cfa29988304f781814#diff-2eb5d63af99d473bf0c208ce9c699740 on whether to use `toBe` for things like this - might be worth having a look at this because we'll probably want to come to a shared consensus.